_posts/2019-12-09-ebcdic-character-format-guide.markdown

```diff
@@ -31,52 +31,35 @@ This post is mainly going to deal with fixed width EBCDIC records.
 
 At it's heart EBCDIC is just a character encoding, in simple terms that is how we map a number onto character as computers only store numbers i.e. 61:a, 62:b, 63:c.
 
-There are multiple different layouts that are inoperable with each other, as they are all slightly different.
+There are multiple different versions of EBCDIC that are inoperable with each other, the different formats are defined by their code pages, and they are all different so you should check which one your project uses.
 
-EBCDIC contains data that has a schema, sort of data in a database record. Each record contains the data for a row in the table.
+Here is an example of a fixed width record (in ASCII for readability).
 
-The schemas are stored in separate files called copybooks, there are two main ways these records can be composed, either fixed or variable width records. The difference between these is how a record is formatted. A fixed width record will always have the exact same length e.g. fixed width of 512 bytes, means each record would be that length even if it was just a name. 
+{% highlight text %}
+00000001Niall     2
+00000002Martin    9
+{% endhighlight %}
 
-PICTURE HERE OF A FIXED WIDTH.?
+As you can see in the data, there is no schema baked into this record, so if you received this file you would not know how to parse it. 
 
-NUMBERS ARE STORED AS CHARs?
+The schemas are stored in separate files called copybooks, there are two main ways these records can be composed, either fixed or variable width records. The difference between these is how a record is formatted. A fixed width record will always have the exact same length e.g. fixed width of 512 bytes, means each record would be that length even if it was just a name. 
```

Comment:
EBCDIC fields are a number of characters wide rather than a number of bytes. It's worth using the same terminology throughout as when we get to explaining copybooks the reader may become confused between bytes and characters.
Add a short paragraph that explicitly states how many bytes in an ASCII and an EBCDIC character, what this means with regards to special character encoding etc. From there you can expand on fixed field widths etc